classes/03class/exercises/c03-docker01/Faria-Marcio/Dockerfile

```diff
@@ -0,0 +1,13 @@
+#Alpine docker image with Terraform
+
+FROM alpine:latest

```

Comment:
You should avoid using the latest tag as you won't know the contents of the latest version. Please, change to a specific version of alpine.